virttest/utils_test/libvirt.py

```diff
@@ -3577,3 +3578,26 @@ def get_disk_alias(vm, source_file=None):
                 logging.info("Ignore error of source attr getting for file: %s" % e)
                 pass
     return None
+
+
+def check_cmd_expected(check_cmd, matched, is_expected):
+    """
+    check the cmd result that process run
+
+    Params check_cmd: the command need to run
+    Params matched: need to match or not match result
+    Params is_expected:
+    """
+    out = decode_to_text(process.system_output(check_cmd,
+                                               ignore_status=False,
+                                               shell=True))
+
+    if matched and not re.search(matched, out):
```

Comment:
here I think the "matched" must be given, so no need to do judgement here

Reply:
better logic is:
```
if is_expected:
    if not re.search():
        raise xxx
    else:
         logging.info()
else:
    if re.search():
        raise xxx
    else:
        logging.info()
```

Reply:
the match check here is that i think some time we may check the result as  "blank"